code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -70,12 +70,71 @@
 			new /obj/item/borg/upgrade/modkit/lifesteal(src)
 			new /obj/item/weapon/bedsheet/cult(src)
 		if(28)
-			new /obj/item/borg/upgrade/modkit/bounty(src)
+			new /obj/item/weapon/disk/design_disk/modkit_disc/bounty(src)
 
+//KA modkit design discs
+/obj/item/weapon/disk/design_disk/modkit_disc
+	name = "KA Mod Disk"
+	desc = "A design disc containing the design for a unique kinetic accelerator modkit."
+	icon_state = "datadisk1"
+	var/modkit_design = /datum/design/unique_modkit
 
+/obj/item/weapon/disk/design_disk/modkit_disc/Initialize()
+	. = ..()
+	var/datum/design/D = new modkit_design
```

Comment:
Could be just `blueprints[1] = new modkit_design`, D is not really being used for anything here.